### PR TITLE
Booking function fixed

### PR DESCRIPTION
**Pull request by giacomofrigo**

### CODE REVIEW

**Comment by vincenzopalazzo** [comment]
When you finish with this PR, ping me that I rebase on top on main @giacomofrigo

File: monolith/services/restaurant_services.py [hunk 1]
```diff
@@ -111,3 +111,23 @@ def get_reservation_rest(owner_id, restaurant_id, from_date, to_date, email):
         # execute and retrive results...
         result = db.engine.execute(stmt, params)
         return result.fetchall()
+
+    @staticmethod
+    def reviewRestaurant(restaurant_id, reviewer_id, stars, review):
```
Comment (vincenzopalazzo) [inline monolith/services/restaurant_services.py:116]: you can use the convention with review_estaurant?

File: monolith/services/restaurant_services.py [hunk 2]
```diff
@@ -111,3 +111,23 @@ def get_reservation_rest(owner_id, restaurant_id, from_date, to_date, email):
...
+        new_review = Review()
+        new_review.restaurant_id = restaurant_id
+        new_review.reviewer_id = reviewer_id
+        new_review.stars = stars
+        new_review.review = review
+
+        db.session.add(new_review)
+        db.session.commit()
+
+        return True
```
Comment (vincenzopalazzo) [inline monolith/services/restaurant_services.py:133]: Can you test this code before joining with the main? I can test it if you have other work to do

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
only a few comment @giacomofrigo

File: monolith/services/restaurant_services.py (hunk 1 above)
Comment (giacomofrigo) [reply monolith/services/restaurant_services.py:116]: ok modifico

File: monolith/services/restaurant_services.py (hunk 2 above)
Comment (giacomofrigo) [reply monolith/services/restaurant_services.py:133]: yes, I have to add the tests!

Comment (vincenzopalazzo) [reply monolith/services/restaurant_services.py]: thanks, i will make the form test

**Review by vincenzopalazzo — APPROVED** [review]
@vincenzopalazzo review this PR and thank to make travis happy

**Pull request merged**